content/blog/2018-06-22-new-editors.md

```diff
@@ -0,0 +1,49 @@
+
+Editors manage the review process, performing initial package checks, identifying reviewers, and moderating the process until the package is accepted by reviewers and transferred to rOpenSci. Anna Krystalli and Lincoln Mullen have both served as guest editors for rOpenSci. As a Research Software Engineer for University of Sheffield RSE, Anna brings the experience of working at the intersection of science and software development that we need. Lincoln has worked broadly in reproducibility for the digital humanities, and led editorial for specialized text analysis packages on an ad-hoc basis. Anna and Lincoln join as full members of the [editorial board](https://github.com/ropensci/onboarding#editors) with Scott Chamberlain, Karthik Ram, Noam Ross and Maëlle Salmon.
+
+<!---<img src="/img/blog-images/2018-06-22-new-editors/anna-krystalli.jpg" alt="Anna Krystalli" style="margin: 0px 20px; width: 250px;" align="left"> _quote from Anna_ --->
+
+<div style="overflow: hidden; width: 100%;">
+<img src="/img/blog-images/2018-06-22-new-editors/anna-krystalli.jpg" alt="Anna Krystalli" style="margin: 15px 25px 25px 0px; width: 250px;" align="left">
+</div>
+
+> quote from Anna
+
+**Anna Krystalli** is a Research Software Engineer at [University of Sheffield](http://rse.shef.ac.uk/) with a PhD in marine macroecology. She has helped organize [Reprohacks](http://rse.shef.ac.uk/blog/opencon_london/) where teams try to reproduce papers nominated by their authors from supplied code and data, and is one of Mozilla Foundation's "50 People Who Made the Internet a Better Place in 2016". Anna reviewed the [`codemetar`](https://github.com/ropensci/onboarding/issues/130) and [`rdflib`](https://github.com/ropensci/onboarding/issues/169) packages for rOpenSci. During her second package review, Anna started work on an experimental reviewing workflow package, [`pkgreviewr`](https://github.com/ropenscilabs/pkgreviewr).
+
+Anna on [GitHub](https://github.com/annakrystalli), [Twitter](https://twitter.com/annakrystalli), [website](http://annakrystalli.me/)
+
+<!---<img src="/img/blog-images/2018-06-22-new-editors/lincoln-mullen.jpg" alt="Lincoln Mullen" style="margin: 0px 20px; width: 250px;" align="left">  _quote from Lincoln_ --->
+
+<div style="overflow: hidden; width: 100%;">
+<img src="/img/blog-images/2018-06-22-new-editors/lincoln-mullen.jpg" alt="Lincoln Mullen" style="margin: 15px 25px 25px 0px; width: 250px;" align="left">
+</div>
+
+> quote from Lincoln
```

Comment:
I don't have anything pithy to say, but you are welcome to edit this quotation however you see fit:

> My software has benefitted from rOpenSci's thorough peer review. I am glad to join the editorial team to help accomplish the rOpenSci mission of making software recognizable as scholarship.